src/store/Slide/ProductsSlide.js

```diff
@@ -8,16 +8,42 @@ const initialState  = {
     filter: {
         _page: 1,
         _limit: 8, 
+    },
+    hot: {
+        filter: {
+            _page: 1,
+            _limit: 8,
+            type: "hot"
+        },
+        list: [],
+        isLoading: false,
+        status: null,
+
     }
 
 }
-
+//Get all products
 export const getProducts = createAsyncThunk(
     'products/getProducts',
     async (filter) => {
         try {
             const paramString = queryString.stringify(filter)
-            const res = await axios.get(`http://localhost:3004/products?${paramString}`)
+            const res = await axios.get(`${process.env.REACT_APP_DATA}/products?${paramString}`)
+            const data = res.data
+            return { data,filter }
+        }
+        catch(err) {
+            return err
+        }
+    }
+)
+// Get hot products
```

Comment:
```suggestion
)

// Get hot products
```
Ngăn cách nhau mỗi block, tự self check code trước khi ready nhé